contracts/multisig/src/multisig_perform.rs

```diff
@@ -164,6 +192,10 @@ pub trait MultisigPerformModule:
             Action::Nothing => OptionalValue::None,
             Action::AddBoardMember(board_member_address) => {
                 self.change_user_role(action_id, board_member_address, UserRole::BoardMember);
+                require!(
```

Comment:
this change will also enforce the change the change of the condition inside the require, since it was initially designed to change the number of board members and to check after if it still meets the requirements

Reply:
for AddProposer the check needs to remain after change_user_role since at that point we are not sure if we are talking about a new user or a current board member becoming a poposer, so we don't know at that moment if num_board_members will increase or will remain unchanged

Reply:
*note to myself - the AddProposer functionality makes no sense - by "Adding a proposer" we are actually "removing the signing right" for the user if he had it previously. This is an unexpected side-effect